pkg/util/loop/loop_linux_test.go

```diff
@@ -11,9 +11,27 @@ import (
 	"syscall"
 	"testing"
 
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
 	"github.com/sylabs/singularity/internal/pkg/test"
+	"github.com/sylabs/singularity/pkg/util/singularityconf"
 )
 
+func getMaxLoopDevices() int {
```

Comment:
This `getMaxLoopDevices` is repeated in many locations - please instead define it once as an exported/Public function (probably in `pkg/util/loop` is a good place - and then call it from other places it is needed.